blueocean-pipeline-api-impl/src/test/java/io/jenkins/blueocean/rest/impl/pipeline/CredentialApiTest.java

```diff
@@ -252,5 +252,18 @@ public void createSshCredentialUsingDefaultSshOnMasterInUserStore() throws IOExc
         Assert.assertEquals("SSH Username with private key", resp.get("typeName"));
         Assert.assertEquals("blueocean-domain", resp.get("domain"));
     }
+    
+    @Test
+    public void createPersonalSSHKey() throws IOException, UnirestException {
+        User user = login();
+
+        Map resp = new RequestBuilder(baseUrl)
+                .status(200)
+                .jwtToken(getJwtToken(j.jenkins,user.getId(), user.getId()))
+                .get("/organizations/jenkins/credentials/user/personalSSHKey").build(Map.class);
```

Comment:
@jtnord i see, this needs to be genericized at the test base class level, I think and applied here, as well as other places